### ISSUE TITLE
.mount units should have separate start & stop timeouts, like services

### ISSUE THREAD

**Issue by lnicola** [systemd/systemd#13871]
**systemd version the issue has been seen with:** 243.78

**Used distribution:** Arch Linux

**Expected behaviour you didn't see:** Unmount jobs on shutdown respect `DefaultTimeoutStopSec` and pressing `Ctrl-Alt-Del` multiple times forces reboot.

**Unexpected behaviour you saw:** 90 seconds hang on shutdown with inaccessible network shares; Ctrl-Alt-Del displays the forced reboot message but does nothing otherwise.

**Steps to reproduce the problem:**

1. mount network share (CIFS via `fstab` in my case)
2. unplug network cable from server
3. reboot

**Probably caused by:** [`umount_with_timeout` using `DEFAULT_TIMEOUT_USEC`](https://github.com/systemd/systemd/blob/adb7b782f83ca076b09b60a905f9ffb311835693/src/shutdown/umount.c#L450) instead of the configured value

**Comment by poettering** [comment]
which umount precisely ar you talking of? can you provide a screenshot of the shutdown phase where you are seeing this, ideally with logging set to debug (`systemd-analyze log-level debug`)

**Comment by lnicola** [comment]
Changing the log level didn't seem to have any effect.

![image](https://user-images.githubusercontent.com/308347/67846972-bd8eae00-fb0a-11e9-954f-dde053a9edaf.png)

**Comment by poettering** [comment]
.mount units currently have no separate start and stop timeouts, they have a single one applied to both operations, and it's initialized by DefaultTimeoutStartSec=.

**Comment by lnicola** [comment]
You're right. `DefaultTimeoutStartSec` seems to work, sorry for the bad analysis.

Is there a way to set it for all mount units with an override in `/etc/systemd` or `x-systemd.mount-timeout=`?

**Comment by bl33pbl0p** [comment]
Related: https://github.com/systemd/systemd/issues/8395

I opened it because of the same usecase: controlling the timeout of unmounting a mount.
